typescript/commands/references.py

```diff
@@ -36,6 +37,23 @@ def is_visible(self):
         # TODO(uforic) hide if text isn't clicked on, imitating Sublime Go to def behavior
         return True
 
+    @staticmethod
+    def __decorate(file_name, line, references):
+        helper = RefsHelper()
```

Comment:
Error handling - not being able to open a file or some weird IO error will be pretty common, so let's just fill the text with "Error retrieving preview," and successfully parse the rest